docs/user-guide/widgets.md

```diff
@@ -67,27 +93,30 @@ Just below the chart's preview, the following configurations are available:
 !!! Note
     The *No operation* option is used when the aggregation method is not needed for the chart. If *No Operation* is selected, no aggregation will be carried out for the chart and the WFS service will be used to generate the chart without using the WPS process `gs:Aggregate` in GeoServer.
 
-* Enable the chart's legend by activating **Display Legend**
+##### Trace Style
 
-* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*) or choose to **Customize the color**.
+The *Trace style* is displayed as follows:
 
-#### Color customization
+<img src="../img/widgets/trace_style.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
-For *Bar Charts* and *Pie Charts*, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) provides the possibility to customize the colors of the charts bars and slices. From the **Color** option dropdown menu, the user can select the *Custom* option and open the **Custom Colors Settings** modal through the <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> button.
+The user can customize the style with the following options choosing between `Simple style` or `Classification style`
 
-<img src="../img/widgets/custom_colors_settings.jpg" class="ms-docimage"/>
+With the **Simple style**, the editor is allowed to customize the `Fill color`, the `Outline color` and the `Outline width`
 
-Inside this modal, the user is allowed to:
+<img src="../img/widgets/trace_simple_style.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
-* Change the default **Color** of bars or slices (depending on the chart type) through the *Color Picker*. This color will be applied for all values for which a *Class Color* has not been configured.
+###### Classification style
 
-<video class="ms-docimage" controls><source src="../img/widgets/custom_colors.mp4"/></video>
+For *Bar Charts* and *Pie Charts*, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) provides the possibility to customize the colors of the charts bars and slices. It is possible to classify the style based on the attributes of the layer. In this case it is possible to select the `Attribute` to use for the classification along with the classification `Method` (*Quantile*, *Equal interval*, *Natural breaks* and *Unique interval*) and select whether to `Sort` the classification by *Attribute X* or *Attribute Y*. Choose a `Color ramp`, the number of classification `Intervals` and the order (with `Reverse order`) of the classification intervals colors. Finally customize the `Outline color` and the `Outline width`.
+
+<img src="../img/widgets/trace_classification_style.jpg" class="ms-docimage"  style="max-width:450px;"/>
```

Comment:
Please use a chart where the stacked values are better represented in a nicer way. For example the one available [here](https://dev-mapstore.geosolutionsgroup.com/mapstore/#/dashboard/21694)

![image](https://github.com/geosolutions-it/MapStore2/assets/1280027/fc8b47cd-e234-4d94-b3c8-9b3b5c06fd28)

Reply:
@ElenaGallo what about this one?

Reply:
Added a new image with commit: [review_9669](https://github.com/geosolutions-it/MapStore2/pull/9677/commits/a4148f53c053eea7453c8fe5387910d2ccc2d122)

Reply:
@ElenaGallo correct if I'm wrong but this change doesn't appear in the doc build:
https://mapstore--9677.org.readthedocs.build/en/9677/user-guide/widgets/

did I miss something?